webpack.config.babel.js

```diff
@@ -34,7 +35,10 @@ const WrapperPlugin = require('wrapper-webpack-plugin');
 
 
 const staticSiteGeneratorConfig = {
-  paths: Object.keys(ROUTES).map(key => ROUTES[key]),
+  paths: [
+    ...Object.keys(ROUTES).map(key => ROUTES[key]),
```

Comment:
Why would the order here matter? This only collects the paths which are covered by SSR. If a path exists twice, that shouldn't make a difference. If this even is relevant needs to be shown once I've solved the SSR problem...